sale_timesheet_order_line_no_update/models/project.py

```diff
@@ -45,4 +45,5 @@ def _compute_sale_line_id_domain(self):
 
     @api.onchange("sale_line_id")
     def _onchange_sale_line_id(self):
-        self.task_ids.write({"new_sale_line_id": self.sale_line_id})
+        for task in self.task_ids:
+            task.new_sale_line_id = self.sale_line_id
```

Comment:
You can do multi-assignation in one shot:

```suggestion
        self.task_ids.new_sale_line_id = self.sale_line_id
```

Reply:
does not work like this

Reply:
@pedrobaeza works with _origin. Please merge.